src/overlays/actors/ovl_En_Hint_Skb/z_en_hint_skb.c

```diff
@@ -143,19 +143,19 @@ typedef enum EnHinkSkbAnimation {
 } EnHinkSkbAnimation;
 
 static AnimationInfo sAnimationInfo[ENHINTSKB_ANIM_MAX] = {
-    { &object_skb_Anim_0064E0, 0.96f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },      // ENHINTSKB_ANIM_0
-    { &object_skb_Anim_003584, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },       // ENHINTSKB_ANIM_1
-    { &object_skb_Anim_002190, 0.6f, 0.0f, 0.0f, ANIMMODE_ONCE_INTERP, 4.0f }, // ENHINTSKB_ANIM_2
-    { &object_skb_Anim_002AC8, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },       // ENHINTSKB_ANIM_3
-    { &object_skb_Anim_00270C, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },       // ENHINTSKB_ANIM_4
-    { &object_skb_Anim_00697C, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // ENHINTSKB_ANIM_5
-    { &object_skb_Anim_006D90, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // ENHINTSKB_ANIM_6
-    { &object_skb_Anim_001D1C, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // ENHINTSKB_ANIM_7
-    { &object_skb_Anim_003584, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // ENHINTSKB_ANIM_8
-    { &object_skb_Anim_003584, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -16.0f },      // ENHINTSKB_ANIM_9
-    { &object_skb_Anim_002AC8, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // ENHINTSKB_ANIM_10
-    { &object_skb_Anim_0015EC, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // ENHINTSKB_ANIM_11
-    { &object_skb_Anim_0009E4, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },       // ENHINTSKB_ANIM_12
+    { &gStalchildWalkAnim, 0.96f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },        // STALCHILD_ANIM_WALK
+    { &gStalchildStandUpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },      // STALCHILD_ANIM_STAND_UP_MORPH_1
+    { &gStalchildAttackAnim, 0.6f, 0.0f, 0.0f, ANIMMODE_ONCE_INTERP, 4.0f }, // STALCHILD_ANIM_ATTACK
+    { &gStalchildStaggerAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },      // STALCHILD_ANIM_STAGGER_MORPH_4
+    { &gStalchildCollapseAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },     // STALCHILD_ANIM_COLLAPSE
+    { &gStalchildSitLaughAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },     // STALCHILD_ANIM_SIT_LAUGH
+    { &gStalchildSitTapToesAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },   // STALCHILD_ANIM_SIT_TAP_TOES
+    { &gStalchildSwingOnBranch, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },    // STALCHILD_ANIM_SWING_ON_BRANCH
+    { &gStalchildStandUpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },      // STALCHILD_ANIM_STAND_UP_MORPH_8
+    { &gStalchildStandUpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -16.0f },     // STALCHILD_ANIM_STAND_UP_MORPH_16
+    { &gStalchildStaggerAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },      // STALCHILD_ANIM_STAGGER_MORPH_16
```

Comment:
```suggestion
    { &gStalchildStaggerAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },      // STALCHILD_ANIM_STAGGER_MORPH_8
```
Since morph is over 8 frames, not 16. I wonder if it could be `MORPH_SLOW` or `MORPH_FAST`, but it's fine as is for me

Reply:
Good catch, I missed that one!